clustered/server/src/main/java/org/ehcache/clustered/server/offheap/OffHeapServerStore.java

```diff
@@ -29,14 +32,29 @@
 public class OffHeapServerStore implements ServerStore {
 
   private final List<OffHeapChainMap<Long>> segments;
+  private final List<ServerStoreEvictionListener> listeners = new CopyOnWriteArrayList<ServerStoreEvictionListener>();
```

Comment:
Why not a single one? Seeing other places, it looks like we stack three collections on top of each other - that seems overkill.